packages/roosterjs-editor-plugins/lib/plugins/Picker/PickerPlugin.ts

```diff
@@ -53,18 +53,65 @@ const UNIDENTIFIED_CODE = [0, 229];
  */
 export default class PickerPlugin<T extends PickerDataProvider = PickerDataProvider>
     implements EditorPlugin {
-    private editor: IEditor;
-    private eventHandledOnKeyDown: boolean;
-    private blockSuggestions: boolean;
-    private isSuggesting: boolean;
-    private lastKnownRange: Range;
+    private editor: IEditor | undefined = undefined;
```

Comment:
I think editor is not required for this class at all. It is only needed in the core class and we have already pass it in when it is constructed.

Reply:
This editor is for the plugin picker class, we pass it in the PickerPluginCore constructor.

Reply:
The fact is, we only get editor instance from initialize() function, then we pass it into core, then we will never use it again. That means the picker class doesn't need to hold it at all.